packages/datepickers/src/elements/DatepickerRange/DatepickerRange.tsx

```diff
@@ -15,37 +15,36 @@ import Calendar from './components/Calendar';
 
 export interface IDatepickerRangeProps {
   locale?: string;
-  /**
-   * The start date to display
-   */
+  /** Sets the start date to display */
   startValue?: Date;
-  /**
-   * The end date to display
-   */
+  /** Sets the end date to display */
   endValue?: Date;
-  /**
-   * Minimum value. Users are still able to manually enter a date below this value.
-   */
+  /** Sets the minimum date value. Note: users can still manually enter a date below this value. */
   minValue?: Date;
-  /**
-   * Maximum value. Users are still able to manually enter a date above this value.
-   */
+  /** Sets the maximum date value. Note: users can still manually enter a date above this value. */
   maxValue?: Date;
   /**
-   * Returns the parsed start and end dates
+   * Handles changes to the start and end dates
+   *
+   * @param {Date} startValue The start date to use
+   * @param {Date} endValue The end date to use
    */
   onChange?: (values: { startValue?: Date; endValue?: Date }) => void;
   /**
-   * Allows customization of date formatting within input.
+   * Formats the date
+   *
+   * @param {Date} date The date to format
+   * @returns {string}
    */
   formatDate?: (date: Date) => string;
   /**
-   * Override default date parsing
+   * Parses the date with a custom format and returns a `Date` object
+   *
+   * @param {string} inputValue The format used for parsing
+   * @returns {Date}
```

Comment:
Reduce the overall description and move the description of the return value to the `@returns` clause.

Reply:
Fixed in latest push.